quakemigrate/signal/pickers/gaussian.py

```diff
@@ -422,19 +431,20 @@ def _find_peak(self, windowed_onset, threshold):
 
         Returns
         -------
-        true_peak_idx : [float, float]
-            Start and end index values for the 'true' peak, with an additional
-            buffer of 1 data point either side.
+        true_peak_idx : [int, int]
+            Start and end index values for the 'true' peak, with +1 added to
+            the last index so that all of the values above the threshold are
+            returned when slicing by index.
 
         Raises
         ------
         util.NoOnsetPeak
-            If no onset data exceeds the threshold.
+            If no onset data, or only a single sample, exceeds the threshold.
 
         """
 
         exceedence = np.where(windowed_onset > threshold)[0]
-        if len(exceedence) == 0:
+        if len(exceedence) < 2:
```

Comment:
Does this do quite what we want? What if there are multiple periods that exceed the threshold in a window, but the 'true' peak is a single sample? Would the test be better in the `_find_peak` method?

Reply:
Yep, good catch. Have addressed this in 692a6e7